src/modm/driver/display/sh1106_i2c.hpp

```diff
@@ -62,14 +59,14 @@ class Sh1106 : public Ssd1306<I2cMaster, Height>
 	{
 		RF_BEGIN();
 		// Default on Power-up - can be omitted
-		this->commandBuffer[0] = ssd1306::AdressingCommands::MemoryMode;
-		this->commandBuffer[1] = ssd1306::MemoryMode::PAGE;
+		this->commandBuffer[0] = uint8_t(ssd1306::AdressingCommands::MemoryMode);
```

Comment:
Since C++23 the new preferred way to convert an enum to its underlying integer type is [`std::to_underlying`](https://en.cppreference.com/w/cpp/utility/to_underlying). It avoids the potential mistake of getting the type wrong. Not so important here and it's more characters. Just for information, I wouldn't really object to `uint8_t`.

Reply:
I wasn't aware of this feature, thanks!
The disadvantage is unfortunately that this makes the code lines even longer...